src/api/cardsApiMock.json

```diff
@@ -0,0 +1,6 @@
+[
```

Comment:
same for the `src/api/boardsApiMock.json` regarding the location of that file

Reply:
Thanks! Actually we don't need this file as the cardsApiMock is not being used so will delete this file.

Reply:
moved